src/module/item/ancestry/document.ts

```diff
@@ -143,6 +147,11 @@ class AncestryPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends
             actor.rollOptions.all[`self:trait:${trait}`] = true;
         }
     }
+
+    /** Generate a list of strings for use in predication */
+    override getRollOptions(): string[] {
+        return [...super.getRollOptions(), `item:rarity:${this.rarity}`];
+    }
```

Comment:
This needs to accept the standard parameter

Reply:
Prefix was added

Reply:
the roll option must then use that prefix

Reply:
new version of code 
`return [...super.getRollOptions(prefix), `item:rarity:${this.rarity}`];`